src/celeritas/phys/PhysicsParams.cc

```diff
@@ -533,7 +534,16 @@ void PhysicsParams::build_xs(Options const& opts,
                 xs_grid_ids[mat_idx] = build_grid(builders[VGT::macro_xs]);
                 eloss_grid_ids[mat_idx]
                     = build_grid(builders[VGT::energy_loss]);
-                range_grid_ids[mat_idx] = build_grid(builders[VGT::range]);
+                if (auto grid_id = eloss_grid_ids[mat_idx])
+                {
+                    //! \todo Use \c spline_eloss_order when enabled by default
+                    size_type constexpr spline_order = 2;
```

Comment:
@amandalund Did you say Geant4 uses cubic splines? Maybe that's the discrepancy here? Perhaps also the boundary treatment (reduced order, extrapolation, ...?) is different